packages/python-packages/doc-warden/warden/WardenConfiguration.py

```diff
@@ -172,22 +179,22 @@ def __init__(self):
     # strips the directory up till the repo root. Allows us to easily think about 
     # relative paths instead of absolute on disk
     def get_output_path(self, input_path):
-        return input_path.replace(os.path.normpath(self.target_directory), '')
+        return input_path.replace(os.path.normpath(self.repo_root), '')
```

Comment:
If a `repo_root` is not present, we should fall back to replacing `self.target_directory`. That would make this a minor version bump.

Reply:
This does make this significantly simpler though. Hmm. I think I'm ok with the change if we bump major version of `doc-warden` here.